ml_service/pipelines/diabetes_regression_build_train_pipeline.py

```diff
@@ -16,81 +16,90 @@ def main():
     aml_workspace = Workspace.get(
         name=e.workspace_name,
         subscription_id=e.subscription_id,
-        resource_group=e.resource_group
```

Comment:
Are there any changes in diabetes_regression_build_train_pipeline.py or diabetes_regression_build_train_pipeline_with_r.py besides formatting?

Reply:
Nope. get_environment() and load_sample_data() are two functions that changed in signature, and are called form these files, but I supplied reasonable defaults so that the call sites in these files do not change at all.